### ISSUE TITLE
Every single preprocessor/model always creates the same invalid annotator result (lots of vertical black and white lines)?

### ISSUE THREAD

**Issue by scottmudge** [Mikubill/sd-webui-controlnet#506]
I've reinstalled Automatic1111's repo probably 3 times now, blank/fresh configs, updated NVIDIA drivers, latest version of Python 3.10, latest version of this plugin.

However, every single time I use ControlNet I get the same invalid annotator result. A bunch of vertical lines:

### Settings/Example: 
![image](https://user-images.githubusercontent.com/19617165/223007181-b28692b8-b75c-4155-90f9-f2e4055b1ac7.png)

### Raw annotator result:
![tmpi5k9sh1e-1](https://user-images.githubusercontent.com/19617165/223007196-2294a534-ec05-4531-9841-97787bb33c9d.png)

Doesn't matter what model or preprocessor I use. Depth produces something slightly different, but still incorrect. No settings seem to change anything.

I've followed numerous basic tutorials exactly, but something seems broken. Can't figure it out. StableDiffusion runs fine except for this extension.

Have tried with `--xformers` and without, no difference. No other runtime options enabled.

Anyone else encounter this? Wanted to check before trying to debug.

**Comment by scottmudge** [comment]
It has the same result with any input image. I left it blank because it had no effect. Scribble, arbitrary input images, blank images... always the same vertical lines:

![image](https://user-images.githubusercontent.com/19617165/223092424-dca39abe-546a-4704-a449-8105e68d7f57.png)

Raw annotator result:

![tmp6z8zwz4x-1](https://user-images.githubusercontent.com/19617165/223092533-66bac70c-8f3f-4f63-bced-d1975ac1d774.png)

Interestingly the *spacing* between the lines changes, but always the same pattern.

Also interestingly the resulting SD image seems to have the lines applied. Result from the prompt above:

![00017-499212416-7-20230306055921](https://user-images.githubusercontent.com/19617165/223092810-a9d6b2f2-a1be-427e-b03f-fe59fb289560.png)

**Comment by jn-jairo** [comment]
What browser are you using? I had a similar problem and it was my browser (Brave) not sending the correct image on the upload, now I am using Google Chrome and it works fine.

**Comment by scottmudge** [comment]
Ah! That was it. Using LibreWolf, which is Firefox based like Brave. 

@Mikubill -- If it's not stated anywhere, you may want to make a note that Firefox based browsers don't appear to work correctly.

**Comment by scottmudge** [comment]
Actually, re-opening. Even with Chromium, the uploaded image doesn't seem to do anything. Scribble seems to work fine (depth and canny produce a valid looking result), but when I drag and upload an image the result is blank. It's not vertical stripes anymore, just white.

**Comment by ljleb** [comment]
Does normal stable diffusion img2img work for you? I'm puzzled by these issues on browser compatibility because we are not doing anything much more complicated than the host webui interface with any of our gradio components.

Maybe separating scribbles and webcam mode in their own sub-tabs could help with this?

**Comment by scottmudge** [comment]
I'll try img2img in a sec, but I'm pretty sure it worked fine.

I *do* have webcam permissions disabled, but I don't use the webcam mode. Does image uploading require webcam permissions for some reason?

**Comment by ljleb** [comment]
Afaik webcam mode is only active when you click on enable webcam button. it and sketch mode should not be a problem, but it's the only thing I'm aware of that we are doing and that could be interferring with browser compatibility...

**Comment by scottmudge** [comment]
Yeah img2img works fine, but uploading the same image to ControlNet yields a blank white annotator result (or fuzzy/blank depth) on Chromium. The sketches/scribbles work though.

It's a step up from the vertical lines Firefox/LibreWolf was causing, but it's indeed strange. Maybe gradio has a compatibility issue.

**Comment by scottmudge** [comment]
Updating that Chrom**ium** does *NOT* work, but *Chrome* does. Weirdly it's failure mode is *different* from Firefox/LibreWolf.

To summarize -- LibreWolf/Firefox causes an annotator result with a bunch of vertical black and white lines. Chrom*ium* just causes a blank white annotator result.

But *Chrome* works as expected.

Same version of Chrome and Chromium.

A little weird... given the only difference between these browsers on my system is that Chromium is "de-googled". Which I prefer.

**Comment by nefclan** [comment]
> Ah! That was it. Using LibreWolf, which is Firefox based like Brave.
> 
> @Mikubill -- If it's not stated anywhere, you may want to make a note that Firefox based browsers don't appear to work correctly.

Might be some privacy add-ons or extensions denying canvas access, like "Brave Shield" from Brave browser [#263](https://github.com/Mikubill/sd-webui-controlnet/issues/263#issuecomment-1438733507), "Canvas Fingerprint Defender" [#426](https://github.com/Mikubill/sd-webui-controlnet/issues/426#issuecomment-1453906396), "Canvas Blocker" in my case.

**Comment by scottmudge** [comment]
Perhaps LibreWolf has something built in, but I was using a completely fresh install (portable). Looks like it does have RFP baked in, which has a lot of anti-fingerprinting features. So that was probably it.

**Comment by Baronco** [comment]
Thanks guys, I use brave and all the results of the preprocessors were black or white images, I disabled brave shield and now everything is fine!
